CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing
+
+Before contributing to this repository, discuss the changes through an issue,
+email, or any mode of communciation.
```

Comment:
This may be a @shaunmaharaj question, but where do they find an email? Where does the email go?